rainfall/templates/page.html

```diff
@@ -1,29 +1,39 @@
-<!-- wp:template-part {"slug":"header","tagName":"header"} /-->
+<!-- wp:group {"backgroundColor":"light-orange","layout":{"type":"default"}} -->
+<div class="wp-block-group has-light-orange-background-color has-background"><!-- wp:template-part {"slug":"header","theme":"rainfall","tagName":"header"} /-->
```

Comment:
remove theme name from the header template part for it to work properly.

![image](https://user-images.githubusercontent.com/1935113/191171834-53a374c4-e570-4a4a-b132-477cef62ea18.png)